### ISSUE TITLE
Not functioning on xfinity 

### ISSUE THREAD

**Issue by Lastcraft484** [smithy212000/PortMiner#29]
https://imgur.com/a/GMfVUxn

Hey, any ideas about what is going on. I have mineos in a freenas jail running the plugin version on spigot 1.16.3. I have tried also just using it on a vanilla server with the executable jar and got the same issue. I have a plex server that uses upnp and that works great so I am not sure what the issue is.

**Comment by smithy212000** [comment]
Have you been running a server that has crashed before?
If so could be a port already open on your router for 25565? If not that, try power cycling your router to see if that pushes anything along but if that doesn't work either I'll take another look.

**Comment by Lastcraft484** [comment]
yeah I tried checking if the port was open or not but it did not open the port at all and power cycling the router diddnt help. Thanks for looking :).

**Comment by smithy212000** [comment]
Have you used PortMiner before and did it work then or is this your first time using it

**Comment by Lastcraft484** [comment]
hm, I got it working now by just using the desktop version. So maybe an issue with my setup or the plugin version

**Comment by smithy212000** [comment]
Could be a firewall running on the NAS, if it's either blocking ports 25565
or 1900 for UPnP, and on your desktop it worked as that firewall doesn't
affect that?
